spec/spec_helper.rb

```diff
@@ -17,6 +17,7 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 require 'factory_girl_rails'
+$LOAD_PATH << File.expand_path(File.join(__FILE__, '../../app/services'))
```

Comment:
Rails should autoload everything under `app/`, you shouldn't have to manipulate `$LOAD_PATH`